docs/reference/rest-api/usage.asciidoc

```diff
@@ -252,6 +252,11 @@ GET /_xpack/usage
     "available" : true,
     "enabled" : true
   },
+  "searchable_snapshots" : {
```

Comment:
Can we have YAML REST tests for this? I know we don't have a good coverage for usage/info REST APIs but I still think it's better than nothing.

Reply:
I've added a test in 0839e24